ouroboros-consensus/src/Ouroboros/Consensus/Protocol/BFT.hs

```diff
@@ -132,6 +133,9 @@ instance BftCrypto c => ConsensusProtocol (Bft c) where
 
   protocolSecurityParam = bftSecurityParam . bftParams
 
+  protocolGenesisWindowLength =
```

Comment:
Might be worth explaining why this is set to a constant. We explain this somewhere else. We might want to reiterate this here, or link to the relevant module.